projects/cashmere/src/lib/sass/navbar.scss

```diff
@@ -226,6 +226,7 @@ $navbar-fixed-shadow: 0px 2px 6px $shadow;
         padding-top: 2px;
     }
     display: flex;
+    line-height: 1.3;
```

Comment:
Thanks Jake! Could we do `1.2` instead of `1.3` on this?  I was just testing it and 1.3 looked a little wide to me.  @corykon - agree/disagree?

Reply:
Sounds good! I'll make that change.

Reply:
Just double checked and 1.2 still allows for descenders like 'y' without it running into the second line